internal/resource/ca_certificate.go

```diff
@@ -97,8 +97,8 @@ func (r CACertificate) Validate(ctx context.Context) error {
 			},
 		}
 	}
-	if !cert.BasicConstraintsValid {
-		errStr := `certificate does not appear to be a CA because` +
+	if !cert.BasicConstraintsValid || !cert.IsCA {
+		errStr := `certificate does not appear to be a CA because ` +
```

Comment:
Misleading error. If the CA attribute is explicitly set to false then the attribute is present (not missing) but set to false. Can we do better?

Reply:
Ouch, good catch! What about this? https://github.com/Kong/koko/pull/495